### PR TITLE
Fix configuration files for the preview + Alignment of the documents with cookiecutter repo

### PR DESCRIPTION
**Pull request by AgostinoPurificato**
**Problem**: `markdown` documents are not visible in the local preview of the site generated with the docker file.

**Solution**: add the dependencies and imports into the configuration files of the preview local site in order to integrate not only `rst` files but also `markdown` documents.

### CODE REVIEW

**Comment by AgostinoPurificato** [comment]
@Clou-dia

**Comment by AgostinoPurificato** [comment]
Added another commit in order to align the content of the documents with the guidelines written into the cookie-cutter repository with a [PR](https://github.com/teamdigitale/dati-semantic-cookiecutter/pull/15)

@Clou-dia

File: docs/04-guida-sviluppo/index.md [hunk 1]
```diff
@@ -40,14 +50,74 @@ Quelle supportate sono:
...
+email: info@teamdigitale.governo.it
+name: Dipartimento per la Trasformazione Digitale
+it:
+  riuso:
+    codiceIPA: pcm
+```
+
+Tutte le risorse fornite DEVONO risiedere all'interno della cartella `assets/`
+referenziato in ndc-config.yaml.
+Le risorse al di fuori du `assets/` non saranno elaborate.
```
Comment (ioggstream) [inline docs/04-guida-sviluppo/index.md]:
```suggestion
Le risorse al di fuori di `assets/` non saranno elaborate.
```

File: docs/04-guida-sviluppo/index.md [hunk 2]
```diff
@@ -40,14 +50,74 @@ Quelle supportate sono:
...
+
+Ogni tipo di asset (ontologie, vocabolari controllati, schemi)
+DEVE risiedere nella sua cartella specifica con un nome predefinito,
+referenziato in ndc-config.yaml.
```
Comment (ioggstream) [inline docs/04-guida-sviluppo/index.md]: Il fatto di mettere i nomi nelle directory nell' `ndc-config.yaml` serviva a gestire la legacy.
Verificate se a regime ha senso, poiché questa feature potrebbe essere abusata.

File: docs/04-guida-sviluppo/redirect.md [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+
+Nel caso del Catalogo, occorre fare riferimento alla [cartella Italia del GIT del w3id](https://github.com/perma-id/w3id.org/tree/master/italia), nella quale DEVONO essere aggiunte le sottocartelle, ciascuna per una particolare area tematica, che conterranno i file di reindirizzamento, eventualmente organizzati in ulteriori sottocartelle, ed il file `README.md`. L’aggiunta delle sottocartelle di `'/Italia'` sarà sottoposta ad approvazione del Comitato Italia; successivamente, le stesse saranno gestite autonomamente e con interfacciamento diretto verso w3id.org dai referenti indicati nei file `README.md`. La cartella e le relative sottocartelle, i  file `.htaccess` e i file `README.md` DEVONO essere creati sul [repository git del w3id](https://github.com/perma-id/w3id.org) dall'Organizzazione contributrice.
```
Comment (ioggstream) [inline docs/04-guida-sviluppo/redirect.md]:
```suggestion
Nel caso del Catalogo, occorre fare riferimento alla [cartella `italia` del repository git del w3id](https://github.com/perma-id/w3id.org/tree/master/italia), nella quale DEVONO essere aggiunte le sottocartelle, ciascuna per una particolare area tematica, che conterranno i file di reindirizzamento, eventualmente organizzati in ulteriori sottocartelle, ed il file `README.md`.
L’aggiunta delle sottocartelle di `'/italia'` sarà sottoposta ad approvazione del Comitato Italia; successivamente, le stesse saranno gestite autonomamente e con interfacciamento diretto verso w3id.org dai referenti indicati nei file `README.md`.
La cartella e le relative sottocartelle, i  file `.htaccess` e i file `README.md` DEVONO essere creati sul [repository git del w3id](https://github.com/perma-id/w3id.org) dall'Organizzazione contributrice.
```

Comment (ioggstream) [reply docs/04-guida-sviluppo/redirect.md]: Se si spezzano le righe lunghe, è possibile diminuire i diff. Questo non dovrebbe modificare il rendering dei file.

File: docs/04-guida-sviluppo/redirect.md [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+
+### Processo di pubblicazione degli htaccess sul w3id
+
+#### fork del repository GIT del w3id.org
+
+Il primo passo per la registrazione dei vari redirect è il fork in locale della
+[cartella Italia del GIT del w3id](https://github.com/perma-id/w3id.org/tree/master/italia). Gli 
+identificativi permanenti (URI) verranno definiti sulla base del percorso nel quale saranno inseriti i vari file `htaccess`. Nel caso del Catalogo, il percorso della cartella “root” per nome-cartella dovrà essere 
+`"italia/<nome-cartella>/"`, dunque il namespace degli URI definiti nella stessa sarà
```
Comment (ioggstream) [inline docs/04-guida-sviluppo/redirect.md]:
```suggestion
`italia/<nome-cartella>/`, dunque il namespace degli URI definiti nella stessa sarà 
```

Dovrebbe bastare il backtick

File: docs/04-guida-sviluppo/redirect.md [hunk 5]
```diff
@@ -0,0 +1,217 @@
...
+
+Il primo passo per la registrazione dei vari redirect è il fork in locale della
+[cartella Italia del GIT del w3id](https://github.com/perma-id/w3id.org/tree/master/italia). Gli 
+identificativi permanenti (URI) verranno definiti sulla base del percorso nel quale saranno inseriti i vari file `htaccess`. Nel caso del Catalogo, il percorso della cartella “root” per nome-cartella dovrà essere 
+`"italia/<nome-cartella>/"`, dunque il namespace degli URI definiti nella stessa sarà 
+`w3id.org/italia/<nome-cartella>/`.
+
+#### aggiunta della cartella
+
+Nel repository in locale creato a partire dalla fork occorrerà creare la cartella `“nome-cartella”` che conterrà l’alberatura di sottocartelle e dei relativi file `.htaccess`, oltre al file `README.md`.
```
Comment (ioggstream) [inline docs/04-guida-sviluppo/redirect.md]:
```suggestion
Nel repository in locale creato a partire dalla fork occorrerà creare la cartella `<nome-cartella>` che conterrà l’alberatura di sottocartelle e dei relativi file `.htaccess`, oltre al file `README.md`.
```

Qui e altrove, usare consistentemente `<nome-cartella>` come fatto sopra.

File: docs/04-guida-sviluppo/redirect.md [hunk 6]
```diff
@@ -0,0 +1,217 @@
...
+
+In tal modo, verranno definiti i seguenti URI:
+-	`w3id.org/italia/<nome-cartella>/controlled-vocabulary`
+-	`w3id.org/italia/<nome-cartella>/data`
+-	`w3id.org/italia/<nome-cartella>/onto`
+
+Il `<nome-cartella>` è molto importante dato che DEVE essere inserito in specifici parametri descritti di seguito nel presente documento, 
+e sarà sempre utilizzato per riferirsi all'insieme delle risorse semantiche del Contributore nell'ambito della configurazione dei file di redirect.
+
+Le regole di redirect associate a ciascun uri DEVONO essere definite nei relativi file `.htaccess` descritti di seguito. Il file `README.md` DEVE contenere i nominativi dei referenti unitamente ai loro riferimenti e-mail e di github. Questi riferimenti DEVONO curare la gestione della cartella e dei relativi file a seguito dell’approvazione di “Italia”. Si DOVREBBE prendere come esempio il file `README.md` sotto la cartella `"/italia"`.
```
Comment (ioggstream) [inline docs/04-guida-sviluppo/redirect.md]:
```suggestion
Le regole di redirect associate a ciascun uri DEVONO essere definite nei relativi file `.htaccess` descritti di seguito. Il file `README.md` DEVE contenere i nominativi dei referenti unitamente ai loro riferimenti e-mail e di github. Questi riferimenti DEVONO curare la gestione della cartella e dei relativi file a seguito dell’approvazione di “Italia”. Si DOVREBBE prendere come esempio il file `README.md` sotto la cartella `/italia`.
```

File: docs/04-guida-sviluppo/redirect.md [hunk 7]
```diff
@@ -0,0 +1,217 @@
...
+Il merge sul branch master verrà effettuato direttamente da w3id.org e determinerà la pubblicazione definitiva dei nuovi URI e relative regole di redirect.
+
+### Contenuto dei file .htaccess e del README.md
+
+Di seguito è data una descrizione di file `.htaccess` per ciascuna tipologia di risorsa, da cui l'Organizzazione contributrice DOVREBBE prendere spunto al fine creare i propri file di redirect. Gli snippet di codice di esempio sono basati sull'assunzione che per ciascuna risorsa semantica (es. ontologia) il nome della stessa nell'URI di referenziazione sia uguale al nome della cartella che contiene i relativi file sul git sorgente, e anche uguale ai nomi dei file a meno dell'estensione. In caso contrario, si renderebbe necessario sviluppare apposite regole di redirect più complesse.
+
+#### controlled-vocabulary
+
+Il file `.htaccess` da inserire nella sottocartella `“italia/nome-cartella/controlled-vocabulary”` DOVREBBE essere creato a prendendo come esempio [quello contenuto nella cartella del GIT “italia/controlled-vocabulary”](https://github.com/perma-id/w3id.org/blob/master/italia/controlled-vocabulary/.htaccess).
```
Comment (ioggstream) [inline docs/04-guida-sviluppo/redirect.md]: <nome-cartella> e senza apici ;)

File: docs/04-guida-sviluppo/redirect.md [hunk 8]
```diff
@@ -0,0 +1,217 @@
...
+
+Esso contiene codice scritto sulla base delle Direttive Apache, e permette di gestire le richieste HTTP in base al valore dell'header Accept e di SYNTAX. A seconda del valore, le URL vengono riscritte in modo diverso o reindirizzate a URL esterni. La specifica azione di riscrittura o reindirizzamento dipende dalla combinazione di Accept e SYNTAX.
```
Comment (ioggstream) [inline docs/04-guida-sviluppo/redirect.md]:
```suggestion
Esso contiene codice scritto sulla base delle Direttive Apache, e permette di gestire le richieste HTTP in base al valore dell'header `Accept` e di SYNTAX. A seconda del valore, le URL vengono riscritte in modo diverso o reindirizzate a URL esterni. La specifica azione di riscrittura o reindirizzamento dipende dalla combinazione di Accept e SYNTAX.
```

File: docs/04-guida-sviluppo/index.md (hunk 2 above)
Comment (AgostinoPurificato) [reply docs/04-guida-sviluppo/index.md]: @Clou-dia

File: docs/04-guida-sviluppo/redirect.md [hunk 9]
```diff
@@ -0,0 +1,217 @@
...
+ 
+Di seguito viene data una descrizione delle direttive di esempio, alle quali DEVONO essere modificati i riferimenti degli URL di atterraggio, oltre all’eventuale modifica/integrazione delle regole al fine di meglio adattarsi al git del Contributore:
+
+<pre>
```
Comment (ioggstream) [inline docs/04-guida-sviluppo/redirect.md]: Che differenza c'è tra `<pre>` e il triplo backtick? 
Se questo non comporta problemi, userei consistentemente la stessa sintassi per tutte le pagine.

File: docs/04-guida-sviluppo/redirect.md [hunk 10]
```diff
@@ -0,0 +1,217 @@
...
+</pre>
+Le precedenti condizioni si applicano solo quando SYNTAX è html. Riscrivono le URL in modo diverso, reindirizzando a URL esterni basati su modelli specifici. DEVONO essere configurati sulla base del nome della cartella tematica con la quale ci si riferisce al particolare insieme di risorse semantiche nel git del w3id.
+
+#### data
+
+Il file `.htaccess` da inserire nella sottocartella “italia/nome-cartella/data” DOVREBBE essere creato a partire da [quello contenuto nella cartella del GIT “italia/data”](https://github.com/perma-id/w3id.org/blob/master/italia/data/.htaccess).
+
+Esso contiene codice scritto sulla base delle Direttive Apache, e permette di configurare il server Apache per consentire l'accesso da qualsiasi dominio alle risorse del server, impostare una variabile di ambiente ROOT_URL con un valore fisso, e quindi riscrivere tutte le richieste in modo che includano ROOT_URL prima dell'URI richiesto. 
+
+Di seguito viene data una descrizione delle direttive di esempio, alle quali DEVONO essere modificati i riferimenti degli URL di atterraggio, oltre all’eventuale modifica/integrazione delle regole al fine di meglio adattarsi al git del Contributore:
```
Comment (ioggstream) [inline docs/04-guida-sviluppo/redirect.md]:
```suggestion
Di seguito vengono descritte delle direttive di esempio, alle quali DEVONO essere modificati i riferimenti degli URL di atterraggio, oltre all’eventuale modifica/integrazione delle regole al fine di meglio adattarsi al git del Contributore:
```

Ove possibile, usare i verbi in modo diretto https://docs.italia.it/italia/designers-italia/writing-toolkit/it/bozza/index.html

File: docs/04-guida-sviluppo/schemi.md [hunk 11]
```diff
@@ -135,7 +137,7 @@ qui una codelist (estensione ".gc") http://publications.europa.eu/resource/distr
 Gli stessi dati possono essere recuperati  a partire da https://data.europa.eu/data/datasets/country.
 
 
-
+(schemi-versionamento)=
```
Comment (ioggstream) [inline docs/04-guida-sviluppo/schemi.md]: :+1:

**Comment by bfabio** [comment]
@AgostinoPurificato @Clou-dia just to be sure: this just syncs [dati-semantic-cookiecutter/CONTRIBUTING.en.md](https://github.com/teamdigitale/dati-semantic-cookiecutter/blob/main/CONTRIBUTING.en.md) into here, right?

As in: if there is anything that doesn't check out for now (and there are a couple things IMO), we can fix that later in the rewrite?

**Comment by AgostinoPurificato** [comment]
@bfabio 
cc: @Clou-dia 
This PR makes some changes on the md files (italian and english version) in order to integrate the redirect's guidelines (in two new md files called `REDIRECT.<...>`) and also for align the existing files to what is written into the cookie-cutter repository. Every other change can be done directly on the draft of the new version of the guide (wip).

File: docs/04-guida-sviluppo/index.md (hunk 2 above)
Comment (bfabio) [reply docs/04-guida-sviluppo/index.md]: Tracked in #19

**Comment by bfabio** [comment]
Merged in #20, my bad

**Review by ioggstream — COMMENTED** [review]
@bfabio late comments, sorry.

**Comment by bfabio** [comment]
@ioggstream thanks, I'm going to open a new issue referencing those comments, so not to lose them when authoring the new draft (https://github.com/teamdigitale/dati-semantic-guida-ndc-docs/pull/15#issuecomment-1831416845)

**Pull request closed without merging**